src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberAnalysisResult.cs

```diff
@@ -79,6 +89,22 @@ public int ParameterFromArgument(int arg)
             return ArgsToParamsOpt[arg];
         }
 
+        public int FirstBadArgument
+        {
+            get
+            {
+                for (int i = 0; i < BadArguments.Capacity; i++)
```

Comment:
>for (int i = 0; i < BadArguments.Capacity; i++) [](http://example.com/codeflow?start=16&length=47)

It looks like ```BitVector.TrueBits``` provides a more efficient way to find "true" bits. If the ```FirstBadArgument``` helper is meant to be used only in error scenarios, we might be fine with allocating the enumerator then. If enumerator's allocation is really a problem for error scenarios as well, we should consider moving this helper to ```BitVector``` and using ```TrueBits``` like way of looking for the answer.

Reply:
I switched to `TrueBits().First()`